src/main.rs

```diff
@@ -75,6 +75,10 @@ struct RootOpts {
     /// Options: `auto`, `always` or `never`
     #[structopt(long)]
     color: Option<Color>,
+
+    /// Watch for changes in configuration file, and reload accordingly.
+    #[structopt(short, long)]
+    watch_config: bool,
```

Comment:
`kubectl` uses it for watching objects.

Reply:
But that's the only major usage that I've found. Other common usage is for `wait`.

There is `-wc` option, but I would pass on that.

Reply:
Yeah, I think `-w` is fine for now :+1:

Reply:
Since this feature isn't available on Windows, should we gate it behind that target so users won't see the option and have it fail when they try to use it?

Reply:
In that case, users would get generic error message like "Unknown `-w` flag" which is less descriptive than the current version. 

But we could/should mention it in documentation on web and cli.